rules/S6496/cfamily/metadata.json

```
@@ -0,0 +1,18 @@
+{
+  "title": "Pass small, trivially-copyable objects by value",
```

```suggestion
  "title": "Small trivially-copyable objects should be passed by value",
```
see https://docs.sonarqube.org/latest/extension-guide/adding-coding-rules/